src/inspector_agent.h

```diff
@@ -39,6 +39,7 @@ class InspectorSessionDelegate {
  public:
   virtual ~InspectorSessionDelegate() = default;
   virtual bool WaitForFrontendMessage() = 0;
+  // XXX SendMessageToFrontend() ?
```

Comment:
I don't think there's ever be a need. And if we manage to make WS transport talk to inspector through the JS API then there will be only one InspectorSessionDelegate implementation, which will remove the need for this interface.